src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Log.cs

```diff
@@ -39,37 +32,19 @@ public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
 
     // Log(number:n|*[n], [base:n|*[n]]):*[n]
     // Equivalent Excel function: Log
-    internal sealed class LogTFunction : BuiltinFunction
-    {
-        public override bool IsSelfContained => true;
+    internal sealed class LogTFunction : MathTwoArgTableFunction
+    {
+        public override bool NonConsistentUseSecondArg => true;
 
         public LogTFunction()
-            : base("Log", TexlStrings.AboutLogT, FunctionCategories.MathAndStat, DType.EmptyTable, 0, 1, 2)
+            : base("Log", TexlStrings.AboutLogT, 1)
         {
-        }
-
+        }
+
         public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
         {
-            yield return new[] { TexlStrings.MathFuncArg1 };
-        }
-
-        public override string GetUniqueTexlRuntimeName(bool isPrefetching = false)
-        {
-            return GetUniqueTexlRuntimeName(suffix: "_T");
```

Comment:
>return GetUniqueTexlRuntimeName(suffix: "_T"); [](http://example.com/codeflow?start=12&length=46)

We still need to keep this override. We'll be removing it soon, but there's some work in Power Apps required to do so

Reply:
I put it back in the base class.  Is that sufficient?